### PR TITLE
update docs to reflect lack of asset holders

### PR DESCRIPTION
**Pull request by geoknee**
⚠️ base not master. Towards #3624

### CODE REVIEW

File: packages/docs-website/docs/contract-api/contract-inheritance.mdx [hunk 1]
```diff
@@ -7,55 +7,36 @@ import Mermaid from '@theme/Mermaid';
 
 The following diagram shows how the contracts inherit from one another. Click on a node to see the corresponding API.
 
-## Assets
+## Assets and Adjudication
```
Comment (geoknee) [inline packages/docs-website/docs/contract-api/contract-inheritance.mdx:10]: Current version(s) viewable at https://docs.statechannels.org/contract-api/contract-inheritance


New version 

Screenshot 2021-07-02 at 16 12 12

File: packages/docs-website/docs/docs/protocol-tutorial/outcomes.md [hunk 2]
```diff
@@ -61,7 +65,7 @@ expect(decodeOutcome(encodedOutcome)).toEqual(outcome);
 
 ## Outcomes that guarantee
 
-Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset holder (say, a given token).
+Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset (say, a given token).
```
Comment (NiloCK) [inline packages/docs-website/docs/docs/protocol-tutorial/outcomes.md:68]:
```suggestion
Guarantee Asset Outcomes are similar to Allocation Asset Outcomes, only they do not have any amounts. Their purpose is to simply express an ordering of destinations for a given asset (say, a given token).
```

this word will have the great joy of living for a little while in this side-trunk.

File: packages/docs-website/docs/docs/protocol-tutorial/release-assets.md [hunk 3]
```diff
@@ -75,33 +20,40 @@ const EOA = ethers.Wallet.createRandom().address;
...
+const stateHash = constants.HashZero; // if the channel was concluded on the happy path, we can use this default value
+const challengerAddress = constants.AddressZero; // if the channel was concluded on the happy path, we can use this default value
+const indices = []; // this magic value (a zero length array) implies we want to pay out all of the allocationItems (in this case there is only one)
 
-/* 
-  Check that an AllocationUpdated event was emitted. 
+const tx2 = NitroAdjudicator.transfer(assetIndex, channelId, outcomeBytes, stateHash, challengerAddress, indices);
+
+/*
+  Check that an AllocationUpdated event was emitted.
```
Comment (NiloCK) [inline packages/docs-website/docs/docs/protocol-tutorial/release-assets.md:43]:
```suggestion
  Check that a FingerprintUpdated event was emitted.
```

**Review by NiloCK — COMMENTED** [review]
Looks good.

**Review by NiloCK — APPROVED** [review]

**Comment by geoknee** [comment]
Thanks @NiloCK, all good suggestions.

**Pull request merged**